pytorch_lightning/callbacks/progress/tqdm_progress.py

```diff
@@ -115,6 +115,26 @@ def __getstate__(self) -> Dict:
         # can't pickle the tqdm objects
         return {k: v if not isinstance(v, _tqdm) else None for k, v in vars(self).items()}
 
+    @property
+    def sanity_check_description(self) -> str:
+        return "Validation Sanity Check"
```

Comment:
This would be more consistent

```suggestion
        return "Sanity Checking"
```